src/main/java/org/quantumbadger/redreader/reddit/prepared/RedditPreparedPost.java

```diff
@@ -712,6 +730,16 @@ public void run() {
 
 		refreshView(activity);
 
+		boolean vote = (action == RedditAPI.RedditAction.DOWNVOTE
+				| action == RedditAPI.RedditAction.UPVOTE
+				| action == RedditAPI.RedditAction.UNVOTE);
+
+		if(src.archived && vote){
+			Toast.makeText(activity, "sorry, this has been archived and can no longer be voted on", Toast.LENGTH_SHORT)
```

Comment:
Strings should be added to the end of strings.xml:

https://github.com/QuantumBadger/RedReader/blob/master/res/values/strings.xml